index.html

```diff
@@ -109,7 +109,7 @@
 <section data-dfn-for="PerformanceServerTiming" data-link-for="PerformanceServerTiming">
   ## The <dfn>PerformanceServerTiming</dfn> Interface
 
-  <pre class='idl'>
+  <pre class='idl' data-cite="hr-time">
```

Comment:
Any IDL block by default searches only for terms in webidl spec. Adding this data-cite tells ReSpec to search for terms in `hr-time` spec also.

Reply:
Is that new?

Reply:
More than a year old I think.. This automatic linking feature is called [`xref`](https://github.com/w3c/respec/wiki/xref)